recipes/plasmapy/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set version = "0.1.2" %}
+
+package:
+  name: plasmapy
+  version: {{ version }}
+
+source:
+  fn: plasmapy-{{version}}.tar.gz
+  url: https://pypi.io/packages/source/p/plasmapy/plasmapy-{{ version }}.tar.gz
+  sha256: 2a2bbc064a8b5726bc32447973babba9355e5c7d9483b1eebc82c6e83a9a173e
+
+build:
+  number: 0
+  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+  skip: True    # [py < 36]
+
+requirements:
+  build:
+    - {{ compiler("c") }}
+    - {{ compiler("cxx") }}
+  host:
+    - python
+    - cython
+    - setuptools
+    - numpy
+    - scipy
+    - astropy
+  run:
+    - python
+    - numpy
```

Comment:
are you compiling against ``numpy`` ?

Reply:
Well, we have a dependency on astropy, and that has a hard dependency on numpy... Also, we have a some Cython scripts that are going to rely on Numpy.

Also, it works for SunPy!